assets/src/edit-story/app/media/mediaProvider.js

```diff
@@ -126,7 +126,11 @@ function MediaProvider({ children }) {
   );
 
   const processor = useCallback(
-    ({ mimeType, posterId, id, src, local }) => {
+    (el) => {
+      if (!el) {
```

Comment:
Alternatively do something like `return accumulatorPromise.then(() => el && processor(el));` below? And let `processor` assume it always gets passed a valid element

Reply:
SGTM. I've added that implementation.